process/block/shardblock.go

```diff
@@ -1170,6 +1256,13 @@ func (sp *shardProcessor) createAndProcessCrossMiniBlocksDstMe(
 		}
 
 		maxTxRemaining := uint32(maxTxInBlock) - nrTxAdded
```

Comment:
move this line before calling the createAndprocessMiniBlockFromHeader function

Reply:
Done